backend/projects/tasks.py

```diff
@@ -48,6 +48,18 @@ def create_tasks_from_dataitems(items, project):
                     continue
                 item[output_field] = item[input_field]
                 del item[input_field]
+        if input_dataset_info.get("copy_from_parent"):
+            # Check if item has a parent
+            if item.get("parent_data"):
+                parent_data = dataset_models.Data.objects.get(id=item["parent_data"])

```

Comment:
Sorry for the dumb question, but where is `dataset_models` coming from inside this file? (I don't see it in the imports, neither inside `projects/models.py` nor `tasks/models.py`)

Reply:
And where is `dataset_models.Data` ?

Reply:
> Sorry for the dumb question, but where is `dataset_models` coming from inside this file? (I don't see it in the imports, neither inside `projects/models.py` nor `tasks/models.py`)

The import is on line 7

Reply:
> And where is `dataset_models.Data` ?

Editing this.